corehq/apps/accounting/migrations/0043_add_web_user_to_payment_methods.py

```diff
@@ -0,0 +1,317 @@
+# -*- coding: utf-8 -*-
+from south.db import db
+from south.v2 import DataMigration
+from django.db import connection
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        sql = """
+            UPDATE accounting_paymentmethod
```

Comment:
Thanks @czue 
I remember another reason I did it this way, which may be unsatisfactory...
```py
PaymentMethod.objects.all().update(web_user=F('billing_admin__web_user"))
```
Which gives me `FieldError: Joined field references are not permitted in this query`
Looking into this further, I found that you can't do joins in F() expressions: https://code.djangoproject.com/ticket/14104

I know this is a small table, so this probably doesn't matter, but I think doing it with raw sql is much more efficient than:

```py
for pm in PaymentMethod.objects.all():
    pm.web_user = pm.billing_admin.web_user
    pm.save()
```
Is there a better way of doing this in Django (I found this: https://github.com/aykut/django-bulk-update) that doesn't call save() on each row separately?

If not, I'm not opposed to changing this code to a loop, just curious if there is a different / more efficient way to handle it.